src/ui/qgsgpsinformationwidgetbase.ui

```diff
@@ -470,7 +470,7 @@ gray = no data
              <item row="17" column="1">
               <widget class="QLineEdit" name="mTxtQuality">
                <property name="toolTip">
-                <string>quality of the position fix: Differential, Non-differential or No position</string>
+                <string>GPS Quality indicator according to NMEA GGA</string>
```

Comment:
Is the term "GPS" correct to use in here? Does that also apply to other positioning systems?

Reply:
```suggestion
                <string>Positioning auality indicator (NMEA GGA or comparable sentence)</string>
```